### PR TITLE
Fixes #24 Fixes #22

### LINKED ISSUE #22: Add apache license to the project
We should add apache 2.0 license to the project.

### LINKED ISSUE #24: Improve ui of art list fragment.
Make some ui improvements in list view. Just make it looks a bit nicer. No requirements yet on it.

### PR DESCRIPTION
**Pull request by genaforvena**
Added cards for list view.
Replaced button with floating button on the explore art activity. It will be on screen always for now as we have to decide strategy when we need to show and hide it.

### CODE REVIEW

**Comment by genaforvena** [comment]
@sanovikov71

File: README.md [hunk 1]
```diff
@@ -7,3 +7,18 @@ Android app to search street art objects near you.
 Any commit will be accepted!
 
 
+---------------------------
+Fuck copyrights! (sort of)
```
Comment (sanovikov71) [inline README.md:11]: Don't like this line :-(

File: app/src/main/res/layout/art_object_in_list.xml [hunk 2]
```diff
@@ -1,30 +1,66 @@
...
-        android:text="Medium Text"
-        android:id="@+id/art_object_view_in_list_name"
-        android:layout_below="@+id/art_object_view_in_list_author"
-        android:layout_toEndOf="@+id/art_object_view_in_list_thumb"/>
-
-</RelativeLayout>
+              android:layout_height="match_parent"
+              android:padding="2dp">
+
+    <android.support.v7.widget.CardView
```
Comment (sanovikov71) [inline app/src/main/res/layout/art_object_in_list.xml:7]: I think that some margins for the cardviews will improve the visual look of the whole list

File: README.md (hunk 1 above)
Comment (genaforvena) [inline README.md:11]: Suggests company name when. It should be a company name.

File: app/src/main/res/layout/art_object_in_list.xml (hunk 2 above)
Comment (genaforvena) [inline app/src/main/res/layout/art_object_in_list.xml:7]: We'll do in the next iteration. This version of list view is not yet final.

**Pull request merged**